release-notes/unreleased/3353-translations-thai.yml

```diff
@@ -0,0 +1,7 @@
+issue_key: 3353
+show_in_stores: true
+platforms:
+  - android
+  - ios
```

Comment:
❓ Why is this not on web?

Reply:
There are no more release notes for web as we only use autogenerated release notes, so I figured I just leave it out. I don't really mind though.